common/src/main/kotlin/com/sourceclear/agile/piplanning/objects/Things.kt

```diff
@@ -71,20 +71,23 @@ sealed class NotificationO(open val id: Long) {
   @JsonTypeName("StoryRequestAccepted")
   data class StoryRequestAccepted(
       override val id: Long,
+      val storyRequestId: Long,
       val sender: String,
       val description: String,
       val notes: String) : NotificationO(id)
 
   @JsonTypeName("StoryRequestRejected")
   data class StoryRequestRejected(
       override val id: Long,
+      val storyRequestId: Long,
       val sender: String,
       val description: String,
       val notes: String) : NotificationO(id)
 
   @JsonTypeName("StoryRequestWithdrawn")
   data class StoryRequestWithdrawn(
       override val id: Long,
+      val storyRequestId: Long,
```

Comment:
requires `storyRequestId` here so that requests with these types do not go to `board/<boardId>/request/undefined/<type>`

Reply:
Seems reasonable, but where does this happen now? Just curious.

Reply:
storyRequestId is required here https://github.com/srcclr/sapling-frontend/blob/master/src/components/NotificationsList.tsx#L19

Reply:
i guess this is no longer necessary since we are going to use the `/acknowledge` path instead.